.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
Just for my curiousity. What does this do?

Reply:
Not sure tbh. It is automatically created by husky per hook (push, commit etc).

https://github.com/typicode/husky
```
npx husky add .husky/pre-commit "npm test"
git add .husky/pre-commit
```

Reply:
Oooo, interesting. And I'm guessing you just add any npm commands you want to be ran pre-push into this file?

Reply:
Yessir